internal/config/funcs/filesystem_test.go

```diff
@@ -206,23 +206,6 @@ func TestFileSet(t *testing.T) {
 			}),
 			false,
 		},
-		{
-			cty.StringVal("."),
-			cty.StringVal("*/*/*.txt"),
-			cty.SetVal([]cty.Value{
-				cty.StringVal("testdata/filesystem/hello.txt"),
-			}),
-			false,
-		},
-		{
-			cty.StringVal("."),
-			cty.StringVal("*/*/hello*"),
-			cty.SetVal([]cty.Value{
-				cty.StringVal("testdata/filesystem/hello.tmpl"),
-				cty.StringVal("testdata/filesystem/hello.txt"),
-			}),
-			false,
-		},
```

Comment:
I deleted these tests cause they were overly dependent on an exact file structure and the functionality was already tested by other cases.